File: mobile/src/main/java/info/papdt/express/helper/services/AppWidgetService.java
package info.papdt.express.helper.services;

import android.content.Intent;
import android.widget.RemoteViewsService;

import info.papdt.express.helper.ui.launcher.ListFactory;

public class AppWidgetService extends RemoteViewsService {

	@Override
	public RemoteViewsFactory onGetViewFactory(Intent intent) {
		return (new ListFactory(this.getApplicationContext(), intent));
	}

}

File: mobile/src/main/kotlin/info/papdt/express/helper/services/AppWidgetService.kt
package info.papdt.express.helper.services

import android.content.Intent
import android.widget.RemoteViewsService

import info.papdt.express.helper.ui.launcher.ListFactory

class AppWidgetService : RemoteViewsService() {

	override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
		return ListFactory(applicationContext, intent)
	}

}
